tests/test_misc.py

```diff
@@ -2101,3 +2103,64 @@ def test_retrieve_vault_status(revault_network, bitcoind):
     mans[0].wait_for_secured_vaults([deposit])
 
     # TODO: same dance with all emergency statuses
+
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_sigfetcher(revault_network, bitcoind, executor):
+    (stks, mans) = revault_network.deploy(2, 1)
+    # First of all, activate a vault
+    vault = revault_network.fund(0.05)
+    revault_network.secure_vault(vault)
+    revault_network.activate_vault(vault)
+
+    # Stopping revaultd
+    for w in stks + mans:
+        w.stop()
+
+    # Deleting the database
+    for w in stks + mans:
+        datadir_db = os.path.join(w.datadir_with_network, "revaultd.sqlite3")
+        os.remove(datadir_db)
```

Comment:
nit: Hmm, you don't really need to loop twice?

Reply:
Right, but I'm keeping "stopping and deleting" and "starting again" as separate loops, I think it's cleaner

Reply:
I don't think so, but i don't really care. Just hearing someone yelling at me INEFFICIENT each time i write an unnecessary loop (which is probably often wrong in Python)